phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableIT.java

```diff
@@ -1352,7 +1352,34 @@ private void assertSequenceNumber(String schemaName, String tableName, long expe
             assertFalse(rs.next());
         }
     }
-    
+
+    @Test
+    public void testAlterTableIfExistsThenOption() throws Exception {
+        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        Connection conn = DriverManager.getConnection(getUrl(), props);
```

Comment:
Use try-with-resources for the connection

Reply:
will do